go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	k8s.io/mount-utils v0.31.3
 	k8s.io/pod-security-admission v0.31.3
 	k8s.io/utils v0.0.0-20240711033017-18e509b52bc8
-	sigs.k8s.io/controller-runtime v0.19.3
+	sigs.k8s.io/controller-runtime v0.19.4
```

Comment:
not sure how this got included, please keep the change to the `userKey` only.

Reply:
Fork was out of sync :)